v3/util/alt_reg_num_ev.go

```diff
@@ -0,0 +1,76 @@
+package util
+
+/*
+* ZLint Copyright 2025 Regents of the University of Michigan
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy
+* of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+* implied. See the License for the specific language governing
+* permissions and limitations under the License.
+ */
+
+import (
+	"github.com/zmap/zcrypto/encoding/asn1"
+)
+
+type RDNSequence []RelativeDistinguishedNameSET
+
+type RelativeDistinguishedNameSET []AttributeTypeAndValue
+
+type AttributeTypeAndValue struct {
+	Type  asn1.ObjectIdentifier
+	Value asn1.RawValue
+}
+
+type parsedSubjectElement struct {
+	IsPresent    bool
+	Value        string
+	Asn1RawValue asn1.RawValue
+	ErrorString  string
+}
+
+type ParsedEvOrgId struct {
+	Rsi, Country, StateOrProvince, RegRef string
+}
+
+func GetSubjectOrgId(rawSubject []byte) parsedSubjectElement {
+	return GetSubjectElement(rawSubject, CabfExtensionOrganizationIdentifier)
+}
+
+func GetSubjectElement(rawSubject []byte, soughtOid asn1.ObjectIdentifier) parsedSubjectElement {
+	result := parsedSubjectElement{IsPresent: false, Value: "", ErrorString: ""}
+	var nl RDNSequence
+
+	rest, err := asn1.Unmarshal(rawSubject, &nl) // parse the sequence of sets, i.e. each list element in nl will be a set
+	if err != nil {
+		return parsedSubjectElement{IsPresent: false, Value: "", ErrorString: "error parsing outer SEQ of subject DN"}
```

Comment:
Would you mind including the original error in the message? It'll help with debugging some day, I am sure of it.